console/src/app/modules/nav/nav.component.html

```diff
@@ -7,12 +7,13 @@
         *ngIf="
           breadc[breadc.length - 1] &&
           !breadc[breadc.length - 1].hideNav &&
-          breadc[breadc.length - 1].type !== BreadcrumbType.AUTHUSER
+          breadc[breadc.length - 1].type !== BreadcrumbType.AUTHUSER &&
+          breadc[breadc.length - 1].type !== BreadcrumbType.INSTANCE
         "
         [ngSwitch]="breadc[0].type"
       >
         <div class="nav-row" @navrow>
-          <ng-container *ngSwitchCase="BreadcrumbType.INSTANCE">
```

Comment:
the customer portal link is added again and a nav item for organizations is added again 
<img width="1398" alt="Screenshot 2024-01-26 at 16 55 58" src="https://github.com/zitadel/zitadel/assets/10165752/2e57e4fb-4393-428a-8401-38047a4040c9">

Reply:
do you i get that correct, that  that using correct service as well as additional filter possibility will be included in #7283 and since this PR only changes the navigation, there won't be any direct conflicts with that, right?

Reply:
correct